src/viem/saga.ts

```diff
@@ -85,15 +81,17 @@ export function* sendPayment({
     feeInfo
   )
 
-  const unlockAndAddStandby = function* () {
+  const unlockWallet = function* () {
     // This will never happen, but Typescript complains otherwise
     if (!wallet.account) {
       throw new Error('no account found in the wallet')
     }
 
     // unlock account before executing tx
     yield* call(unlockAccount, wallet.account.address)
+  }
 
+  const addPendingStandbyTransaction = function* (hash: string) {
```

Comment:
In theory the `addStandbyTransaction` action could be called mutliple times, because it's wrapped in

https://github.com/valora-inc/wallet/blob/ec4623ecc279f2d609d9dfb1c40f442d67fa231f/src/transactions/send.ts#L272

However, right now this `TX_NUM_TRIES` is set to 1.

I think just to be safe, we should update the reduce to replace any standby TX which has the same hash or context id

https://github.com/valora-inc/wallet/blob/b99b375d0a52c2c28ee2bddeaff4c3f529e501eb/src/transactions/reducer.ts#L72-L83

What do you think?

Reply:
Sounds good!